.eslintrc

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "standard"
+  "extends": "standard",
+  "parserOptions": {
+    "sourceType": "script"
+  },
+  "rules": {
+    "strict": [
+      "error"
+    ]
+  }
```

Comment:
@develar per http://eslint.org/docs/user-guide/configuring#configuration-file-formats, eslint supports `package.json` - if some editor only supports `.eslintrc`, then it needs to be fixed.